crates/turbopack-dev-server/src/source/static_assets.rs

```diff
@@ -71,24 +80,29 @@ impl Introspectable for StaticAssetsContentSource {
         let dir = self.dir.read_dir().await?;
         let children = match &*dir {
             DirectoryContent::NotFound => Default::default(),
-            DirectoryContent::Entries(entries) => entries
-                .iter()
-                .map(|(name, entry)| {
-                    let child = match entry {
-                        DirectoryEntry::File(path) | DirectoryEntry::Symlink(path) => {
-                            IntrospectableAssetVc::new(SourceAssetVc::new(*path).as_asset())
-                        }
-                        DirectoryEntry::Directory(path) => StaticAssetsContentSourceVc::new(
-                            format!("{prefix}{name}", prefix = self.prefix),
-                            *path,
-                        )
-                        .into(),
-                        DirectoryEntry::Other(_) => todo!("what's DirectoryContent::Other?"),
-                        DirectoryEntry::Error => todo!(),
-                    };
-                    (StringVc::cell(name.clone()), child)
-                })
-                .collect(),
+            DirectoryContent::Entries(entries) => {
```

Comment:
can we use an early return for `NotFound` to reduce the indent here?

Reply:
Done.